test_scripts/API/PixelDensityAndScale/001_Get_VideoStreamCapability_with_scale_pixelPerInch_diagonalScreenSize_SUCCESS.lua

```diff
@@ -0,0 +1,42 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:
+-- https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0179-pixel-density-and-scale.md
+-- Description:
+-- In case:
+-- 1) During start HMI provides VideoStreamingCapability for parameters: "diagonalScreenSize, pixelPerInch, scale"
+-- 2) Mob app sends GetSystemCapability request to SDL
+-- SDL does:
+-- 1) send response videoStreamingCapability to Mobile with value for "scale, pixelPerInch, diagonalScreenSize"
```

Comment:
videoStreamingCapability is not response, so phrase must be the next:
`send response with videoStreamingCapability`

The same problem in 005-011 scripts